platform/android/app/src/main/kotlin/coop/polypoly/polypod/features/FeatureStorage.kt

```diff
@@ -51,8 +53,12 @@ object FeatureStorage {
                 continue
             }
             val categoryId = FeatureCategory.valueOf(rawCategory.id)
-            val features: MutableList<Feature> = ArrayList()
+            if (rawCategory.visible == false) {
```

Comment:
Android Studio wanted this, I would have preferred `!(rawCategory.visible ?: true)`, like we have on iOS, seems more clear what the default is. But not anything it's IMHO worth delaying a change for anyway, this logic should anyway not live in platform specific code forever.

Reply:
Not very sure about how this is handled in Kotlin, but any chance that `null` will be handled as `false` here?

Reply:
Nope, tested it.